.github/workflows/RELEASE.yaml

```diff
@@ -269,10 +335,10 @@ jobs:
             connectors-bundle-templates-${{ github.event.release.tag_name }}.zip
 
   helm-deploy:
-    needs: build-and-push
+    needs: [maven-release, docker-release]
```

Comment:
Do we need to wait for the `maven-release`? The integration tests usually only require a docker image.